app/_src/gateway/plugin-development/wasm.md

```diff
@@ -1,29 +1,28 @@
 ---
-title: WebAssembly
+title: Create a Proxy-Wasm filter
 book: plugin_dev
 chapter: 11
 ---
 
-# Your First Proxy-Wasm Filter
+## Your first Proxy-Wasm filter
 
-Filters can be written in almost any language that supports WebAssembly as a
-compilation target, but the easiest and most ergonomic way to get started is by
+Filters can be written as a compilation target in almost any language that supports WebAssembly, but the easiest way to get started is by
 using a language which has an existing [Proxy-Wasm](https://github.com/proxy-wasm/spec)
 SDK, such as [Go](https://github.com/tetratelabs/proxy-wasm-go-sdk/)
 or [Rust](https://github.com/proxy-wasm/proxy-wasm-rust-sdk/).
 
 ## Coding, building and deploying a Proxy-Wasm filter
 
-Kong made available Go and Rust Proxy-Wasm filter templates, which can be used
-to start coding your own filters or build as they are and have your first "hello
-world"-type filter:
+Kong made Go and Rust Proxy-Wasm filter templates available, they can be used
+to start writing your own filters or builds as they are and write your first `hello
```

Comment:
I think "build" and "have" were correct before. It means something along the lines of "if you just follow the steps in the template and do nothing, you will get a hello-world filter".

Probably a comma after "your own filters" should be added.